### PR TITLE
Support is_opening/closing when moving position

### PR DESCRIPTION
**Pull request by tetienne**
Fix https://github.com/iMicknl/ha-tahoma/issues/459

### CODE REVIEW

File: custom_components/tahoma/cover.py [hunk 1]
```diff
@@ -317,20 +317,50 @@ async def async_my(self, **_):
...
+        if self.is_awning():
+            # TODO
+            return False
+
+        if self.select_state(CORE_CLOSURE_STATE) > self.select_state(
+            CORE_TARGET_CLOSURE_STATE
+        ):
+            return True
+
+        return False
```
Comment (tetienne) [inline custom_components/tahoma/cover.py:339]: Quick and dirty, need to be clean.

**Comment by tetienne** [comment]
@iMicknl It’s ok for you?

**Review by vlebourl — APPROVED** [review]

File: custom_components/tahoma/cover.py [hunk 2]
```diff
@@ -317,19 +317,41 @@ async def async_my(self, **_):
...
         """Return if the cover is opening or not."""
-        return any(
+        if any(
             execution.get("deviceurl") == self.device.deviceurl
             and execution.get("command_name") in COMMANDS_OPEN + COMMANDS_OPEN_TILT
             for execution in self.coordinator.executions.values()
+        ):
+            return True
+
+        is_moving = self.device.states[CORE_MOVING_STATE].value
```
Comment (iMicknl) [inline custom_components/tahoma/cover.py:327]: Do we still need the loop through current executions, when we can look at the state as well?

Comment (tetienne) [reply custom_components/tahoma/cover.py:327]: I thought about it too. But I don’t know how it will work for the tilt. It’s why I keep it like this.

**Review by iMicknl — APPROVED** [review]

**Pull request merged**